mocha-client-tests/README.md

```diff
@@ -0,0 +1,13 @@
+#Mocha client tests for WebExtensions
+##Install dependency
+`npm install` and `bower install`  
+
+And inside of addon please run `npm i`
+
+##Run with web-ext cli
+If you have it already run `npm run web-ext` if not `npm i web-ext -g` and then first command (will work with FF dev edition), if you have error with web-ext cli please add path for FF binary file with `--firefox-binary /path/to/firefox-bin`
```

Comment:
`npm install -g web-ext` is not necessary because you've already put it in the `devDependencies`. When you do `npm run` it figures out the path to `web-ext` automatically.